lib/modules/dosomething/dosomething_signup/includes/SignupTransformer.php

```diff
@@ -130,6 +130,7 @@ private function setFilters($parameters) {
       'count' => (int) $parameters['count'] ?: 25,
       'page' => (int) $parameters['page'],
       'competition' => $parameters['competition'] ?: NULL,
+      'runs' => dosomething_helpers_format_data($parameters['runs']),
```

Comment:
@deadlybutter you use  `'run_nid' => (int) $parameters['run_nid'],`, maybe I should try this too?